### ISSUE TITLE
About training train_i3d.py

### ISSUE THREAD

**Issue by padimnambawan** [dxli94/WLASL#76]
Is this how it should really be? Please bear with me as I am a beginner in this field. Thank you so much for understanding!

batch size is 6 and learning rate of 0.0001

These are my results after two epochs:

1st: VALIDATION: test Loc Loss: 0.0043 Cls Loss: 0.0043 Tot Loss: 0.0043 Accu :0.0014 

2nd: VALIDATION: test Loc Loss: 0.0043 Cls Loss: 0.0043 Tot Loss: 0.0043 Accu :0.0010
